website/docs/patterns/objects.md

```diff
@@ -95,3 +95,7 @@ Change the instantiated object class and override values from the command line:
 $ python my_app.py db=postgresql db.params.password=abcde
 PostgreSQL connecting to localhost with user=root and password=abcde and database=tutorial
 ```
+
+In addition to instantiating objects from classes, the `instantiate` method can also
+be used for class and static methods.  Simply put the path to the method in the `cls`
+field and use `params` if needed and instatiate your object exactly like a normal class.
```

Comment:
Please describe each of the use cases in more details.
Something along those lines (suggestion, feel free to diverge):

#### class names
An object of that class will be instantiated and returned. passing the config arguments to the `__init__` method
Example class:
```python
class Foo:
  def __init__(x: int, y: int) -> None:
    self.x = x
    self.y = y
```
Example usage:
config:
```yaml
object:
  cls: Foo
  params:
    x: 10
    y: 20
```
```python
result = hydra.utils.instantiate(cfg.object)
```

#### static_method
TODO: similar small example

#### class method
TOOD: similar small example